src/main/groovy/com/vherasymenko/avro/encoder/core/AvroEncoderPort.groovy

```diff
@@ -0,0 +1,11 @@
+package com.vherasymenko.avro.encoder.core
+
+import org.apache.avro.Schema
+
+/**
+ * The avro encoder port.
+ */
+interface AvroEncoderPort {
+
+    String encodeEvent(Schema schema, def avroData, Class classType )
```

Comment:
Why are we using `def` as an input type?  Normally we don't do that.  Does Avro require this somehow?